ccip-sdk/src/chain.ts

```diff
@@ -295,7 +332,7 @@ export abstract class Chain<F extends ChainFamily = ChainFamily> {
    * @param commit - CommitReport range (min, max).
    * @param opts - Optional parameters (e.g., `page` for pagination width).
    */
-  abstract fetchAllMessagesInBatch<
+  abstract getAllMessagesInBatch<
```

Comment:
Nit comment , maybe better to name it getMessagesInbatch (All is kind of redundant)